src/main/java/com/force/i18n/LanguageConstants.java

```diff
@@ -134,6 +134,9 @@ public final class LanguageConstants {
     public static final String CHINESE_SG = "zh_SG"; //Chinese (Simplified) Singapore
     public static final String CHINESE_MY = "zh_MY"; //Chinese (Simplified) Malaysia
     public static final String CHINESE_HK = "zh_HK"; //Chinese (Traditional) Hong Kong
+    public static final String CHUJ = "cac"; //Chuj
```

Comment:
(Hey @yoikawa is this correct?)
Only introduced "Languages" for Mayan (cac, quc, cak) for grammaticus. The "Locales" (cac_MX, cac_GT, quc_GT, cak_GT) will be introduced in sfdc-i18n and Core changes.

Reply:
Hey @yoikawa this might be mainly the question for sfdc-i18n, but in CLDR I could just find the data for Kiche (e.g. https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-localenames-full/main/quc/languages.json#L12 and https://github.com/unicode-org/cldr-json/tree/main/cldr-json/cldr-dates-full/main/quc)

Any comments about Chuj and Kaqchikel? What would be the related locales for them in CLDR (cac, cac_GT, cac_MX or quc, quc_MX seems not to be there)?

Reply:
as long as `Locale` and `ULocale` works, we're fine. and your'e correct.  no change is required for `en-IT` in Grammaticus